packages/react-cosmos-webpack/src/__tests__/webpack-config.js

```diff
@@ -57,8 +59,32 @@ describe('without hmr', () => {
     webpackConfig = getWebpackConfig(userWebpackConfig, cosmosConfigPath);
   });
 
-  test('keeps user loaders', () => {
-    expect(webpackConfig.loaders).toBe(userWebpackConfig.loaders);
+  test('choose `loaders` or `rules` from user config', () => {
```

Comment:
Please move this test outside of the [`without-hmr` describe block](https://github.com/nik-kor/react-cosmos/blob/7abc2236f024217690c6dcfa0b2e09004263ae40/packages/react-cosmos-webpack/src/__tests__/webpack-config.js#L51) as it doesn't benefit from the beforeEach setup. Best to put it at the end of the file.